### ISSUE TITLE
doesn't work when strings contain double-quotation marks

### ISSUE THREAD

**Issue by orzechowskid** [coston/json-to-csv-export#40]
when I try this snippet in the live demo at https://json-to-csv-export.coston.io/ :
```
  <button onClick={() => csvDownload({ delimiter: ',', data: [{ foo: '"hello"', bar: '"there"' }] })}>
    Download Data
  </button>
```
I receive this .csv file:
```
foo,bar
"\"hello\"","\"there\""
```

I expected to receive this .csv file:
```
foo,bar
""hello"",""there""
```
or this one:
```
foo,bar
'"hello"','"there"'
```

I believe this is due to using `JSON.stringify` to encode fields for output, which encloses each field in double-quotes and escapes double-quotes in the data itself with a backslash (which I don't think is correct according to RFC-4180).

**Comment by coston** [comment]
Hey @orzechowskid, thank you for raising this issue. You are right, it's not right that it escapes the input in the CSV file. Would you like to work on a PR to fix the issue?

**Comment by orzechowskid** [comment]
sure, I can try to put something together tonight.  a couple of questions:
- how would you expect to see objects serialized?  (`JSON.stringify` might actually be a good solution here)
- how would you expect to see Dates serialized?  seconds since UNIX epoch?  ISO-8601 strings?  something else?
- do you feel like the answers to these questions should be configurable, or is that too much complexity?

**Comment by coston** [comment]
Hey @orzechowskid, we don't want to worry about date. This package should only be concerned with converting JSON strings to CSV. Other data types should be handled by the user's own data prep for export.

**Comment by orzechowskid** [comment]
opened #42 .

**Comment by coston** [comment]
@orzechowskid your fix is merged and available at `json-to-csv-export@2.1.1`. Thank you for your contribution and your patience. 🚀
